README.md

```diff
@@ -91,11 +98,19 @@ In order to run this project you need:
 - GitHub: [@Atif-Zada](https://github.com/AtifZada)
 - LinkedIn: [@Atif-Zada](https://linkedin.com/in/AtifZada)
 
-<!-- FUTURE FEATURES -->
 
+👤 **Spencer Okyere**
+
+- GitHub: [@skabeo](https://github.com/skabeo)
+- Twitter: [@black_okyere](https://twitter.com/black_okyere)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/spencer-okyere)
 ## 🔭 Future Features <a name="future-features"></a>
 
-- [ ] **[Advance Features ]**
+Possible future features for this project include:
+
+- Add more methods to the `OOP library`
+- Enhance error handling and edge case scenarios
+
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- CONTRIBUTING -->
```

Comment:
![image](https://github.com/Atifzada/OOP_School_Library/assets/93039359/1aad45d0-5016-4dc8-a619-6646685d4e35)

- [ ] It would be nice if you rename this folder from test to spec because when you run rspec then by default rspec looks for the a spec folder so you have to manually tell rspec that check the test folder so kindly rename test folder to spec folder to get out of this conflict 👍

Reply:
Sure I will change it